index.js

```diff
@@ -11,11 +11,21 @@ function toObject(val) {
 	return Object(val);
 }
 
-function isFunc(val) {
-	return typeof val === 'function';
+function assignKey(to, from, key) {
+	var val = from[key];
+
+	if (typeof val === 'undefined' || val === null) {
```

Comment:
You are right, since ECMAScript 5th, `undefined` is not writable anymore :smile: